react/locales/pt-BR.json

```diff
@@ -4,5 +4,7 @@
   "editor.product-details.maxImages.title": "Número máximo de imagens em miniatura",
   "editor.product-details.displayVertically.title": "Exibir miniaturas na vertical",
   "button-label": "Comprar",
+  "addToCartButton.label": "Adicionar ao carrinho",
+  "share.label": "COMPARTILHE",
```

Comment:
Maybe its better use regular string and use text-transform to put it on all uper-case

Reply:
Na verdade nao eh parte do que mudei mas concordo, vou mudar isso.